docs/content/users/quickstart.md

```diff
@@ -339,66 +339,187 @@ DDEV comes ready to work with any PHP project, and has deeper support for severa
 
     In the examples above, we used a one-liner to copy `.env.example` as `env` and set the `DB_HOST`, `DB_DATABASE`, `DB_USERNAME` and `DB_PASSWORD` environment variables to the value of `db`.
 
+    These DDEV’s default values for the database connection.
```

Comment:
Looks like you reintroduced a lot of old stuff? Please take another look. Make sure you're not affecting anything but the Craft section.

Reply:
Yeah, sorry, on it!

Reply:
Truly baffling—not sure how I managed to reintegrate all that. I think it's resolved, aside from the messy commit history. 🙃

Reply:
Messy commit history doesn't matter at all as it will be squash-merged. 

Which one line do you think doesn't belong?

Reply:
It looks like [this](https://github.com/drud/ddev/pull/4197/files#diff-cdacb5ab99042792ddb67ee7d62ac1d36c4699992949005880517b4955e10f7eR322-R335) simplified the list of commands, rendering this sentence unnecessary:

> In the examples above, we used a one-liner to copy `.env.example` as `env` and set the `DB_HOST`, `DB_DATABASE`, `DB_USERNAME` and `DB_PASSWORD` environment variables to the value of `db`.

Reply:
Thanks, yes, that's obsolete.